google-cloud-logging/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+### 1.8.0 / 2019-10-03
+
+#### Features
+
+* Support additional fields of LogSink
```

Comment:
Should this specify that this is on the low-level API, not the public API?

Reply:
SGTM